var/spack/repos/builtin/packages/py-xgboost/package.py

```diff
@@ -29,10 +30,10 @@ class PyXgboost(PythonPackage):
     variant("dask", default=False, description="Enables Dask extensions for distributed training.")
     variant("plotting", default=False, description="Enables tree and importance plotting.")
 
-    for ver in ["1.3.3", "1.5.2", "1.6.1"]:
+    for ver in ["1.3.3", "1.5.2", "1.6.1", "1.6.2"]:
         depends_on("xgboost@" + ver, when="@" + ver)
 
-    depends_on("python@3.7:", when="@1.6:", type=("build", "run"))
+    depends_on("python@3.7:", when="@1.6.1:", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("python@3.7:", when="@1.6:", type=("build", "run"))
```
1.6.0 also requires 3.7+